protobuf/grpc_service.proto

```diff
@@ -1142,6 +1142,12 @@ message ModelRepositoryParameter
     //@@       A string parameter value.
     //@@
     string string_param = 3;
+
```

Comment:
You must also add this to kfserve copy here: https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L276

Reply:
I don't think it is needed, the kfserve one only covers the basic protocol (infer), whereas this is the Triton specific extension. The protobuf message is added within Triton domain

Reply:
I agree with Guan. If this was `InferParameter`, it should be updated in KServer. But `ModelRepositoryParameter` is Triton-specific

Reply:
Ahh.. That's correct. Missed that point.